addons/mail/models/mail_thread.py

```diff
@@ -1986,6 +1993,10 @@ def message_notify(self, *,
         MailThread._notify_thread(new_message, values, **notif_kwargs)
         return new_message
 
+    def _message_log_with_view(self, views_or_xmlid, **kwargs):
```

Comment:
Could be great to add a test in test_mail/tests/test_message_post (we have a ``test_post_log`` test function, could be put right after it)